tests/journey/test_chamber_of_deputies_dataset.py

```diff
@@ -20,10 +20,9 @@ def setUp(self):
     def tearDown(self):
         rmtree(self.path, ignore_errors=True)
 
-    @skip('Takes to long to run')
     def test_fetch_translate_clean_integration(self):
         self.subject.fetch()
-        files = ["Ano-{}.csv".format(n) for n in self.years]
+        files = ["Ano-{}.csv".format(n) for n in [2017]]
```

Comment:
What about `self.years[-1:]`?

Reply:
This was just a attempt to make this test run without use all the resources available in Travis CI and break the build because of it. Looks that worked because it's breaking for a different reason now.